book/src/graffiti.md

```diff
@@ -49,10 +49,12 @@ Below is an example of the validator_definitions.yml with validator specific gra
 ### 3. Using the "--graffiti" flag on the validator client
 Users can specify a common graffiti for all their validators using the `--graffiti` flag on the validator client.
 
+Usage: `lighthouse vc --graffiti emilywest`
```

Comment:
```suggestion
Usage: `lighthouse vc --graffiti example`
```

What is the significance of `emilywest`?

Reply:
nothing